### ISSUE TITLE
Getter `is_alive` is failing

### ISSUE THREAD

**Issue by allarm** [nornir-automation/nornir#493]
Getter `is_alive` is not working
```python
def get_facts(task, napalm_get_bar):

    r = task.run(
        name=f"Getting facts [ {task.host} ]",
        task=napalm_get,
        getters=["is_alive"],
    )

    return r
```

```
Traceback (most recent call last):
  File "/Users/allarm/.virtualenvs/p3_venv/lib/python3.7/site-packages/nornir/core/task.py", line 85, in start
    r = self.task(self, **self.params)
  File "/Users/allarm/.virtualenvs/p3_venv/lib/python3.7/site-packages/nornir/plugins/tasks/networking/napalm_get.py", line 60, in napalm_get
    method = getattr(device, getter)
AttributeError: 'IOSDriver' object has no attribute 'get_is_alive'
```
Because:
```python
getter = g if g.startswith("get_") else "get_{}".format(g)
```
in `/nornir/plugins/tasks/networking/napalm_get.py`

I assume (but haven't checked) that the other getters, not starting from `get_` (the `ping` and `traceroute`) will also fail.

This fixes the issue:
```python
getter = g if (g.startswith("get_") or g in ["ping","is_alive","traceroute"]) else "get_{}".format(g)
```
But having the hardcoded list of getters doesn't seem to be the best solution. Is there a way to get this list from napalm?

**Comment by dbarrosop** [comment]
> But having the hardcoded list of getters doesn't seem to be the best solution. Is there a way to get this list from napalm?

You could check if the driver has the attribute. For instance:

```
if hasattr(driver, getter_name):
     getter = getattr(driver, getter_name)
elif hasattr(driver, f"get_{getter})":
     getter = getattr(driver, f"get_{getter})")
else:
     raise ValueError(f"couldn't find getter {getter_name}")
```

**Comment by ktbyers** [comment]
`is_alive` is not a getter, you can just make a custom task that calls is_alive if you want to use it:

```
def direct(task):
    napalm = task.host.get_connection("napalm", task.nornir.config)
    alive = napalm.is_alive()
```

**Comment by allarm** [comment]
@ktbyers I'm confused: https://napalm.readthedocs.io/en/latest/support/index.html#getters-support-matrix - it clearly states here that `is_alive`, `ping` and `traceroute` are getters. Am I looking at a wrong place? This matrix was the only place with a list of all supported getters I was able to find at all, what am I missing here?

**Comment by ktbyers** [comment]
@allarm Yes, that documentation probably needs updated/fixed (it's auto-built so it is probably a bit hard to separate them). The getters are the methods that start with get_ and relate to retrieving show/operational information from the device. This behavior is also consistent with what we did in napalm-ansible.

Probably easiest fix for is_alive, traceroute, ping is to just use the code I posted earlier.
